dali/kernels/imgproc/sampler.h

```diff
@@ -73,10 +73,8 @@ struct Sampler<DALI_INTERP_NN, In> {
   DALI_HOST_DEV T at(
       int x, int y, int c,
       BorderClamp) const {
-    x = clamp(x, 0, surface.width - 1);
-    y = clamp(y, 0, surface.height - 1);
-
-    return ConvertSat<T>(surface(x, y, c));
+    ivec2 clamped = clamp(ivec2(x, y), ivec2(0, 0), surface.size);
```

Comment:
`surface.size.x` is `surface.width`, while previously it was clamped to `surface.width-1`.

Reply:
Nice catch.